CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ None
 None
 
 #### Bugfixes
-None
+- Fix initial `cornerSide` rendering issue.
```

Comment:
We always put the issue or PR there for reference. like `[#367](https://github.com/IBAnimatable/IBAnimatable/pull/367)`